### DISCUSSION TITLE
江苏地区依然无法使用

### DISCUSSION THREAD

**Discussion by CAB233** [natfrp/wiki#317] [使用问题] [answered]
root@X-WRT:/sbin# natfrpc
2023/05/08 14:28:34 [I] Checking update...
2023/05/08 14:28:37 [I] frpc version: 0.45.0-sakura-3.1 (built: 2023-04-07 12:29)
2023/05/08 14:28:37 [I] connecting [frp-boy.top, tcp]
2023/05/08 14:29:36 [W] login to server failed: read tcp 10.99.42.79:43382->43.249.193.167:7000: read: connection reset by peer
2023/05/08 14:30:43 [W] login to server failed: read tcp 10.99.42.79:38224->43.249.193.167:7000: read: connection reset by peer
2023/05/08 14:31:52 [W] login to server failed: read tcp 10.99.42.79:43434->43.249.193.167:7000: read: connection reset by peer
2023/05/08 14:33:09 [W] login to server failed: EOF
2023/05/08 14:34:17 [W] login to server failed: read tcp 10.99.42.79:51724->43.249.193.167:7000: read: connection reset by peer
2023/05/08 14:35:24 [W] login to server failed: read tcp 10.99.42.79:51236->43.249.193.167:7000: read: connection reset by peer

江苏校园网，在openwrt中使用frpc依然无法连接。我发现命令行中的时间与实际时间不符，简单排查后应该不是时间同步问题。

**Comment by fakeboboliu** [comment, marked as answer]
您好，

这个情况是因为链接被切断，我们无法确定此原因来自于您的 ISP 还是网络管理员。
通常此问题为 校园网/公司网 的安全措施。
